README.md

```diff
@@ -1,75 +1,81 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> Description the project.
+## TODO LIST WEB_APP
 
+- This is a basic website that allows users to add/remove todo's from a list.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
 
-## Live Demo (if available)
+- CSS
 
-[Live Demo Link](https://livedemo.com)
+- Javascript(ES6)
 
+- WebPack
 
-## Getting Started
+- Lighthouse (An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more).
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+- Webhint (A customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors).
 
+- Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
 
-To get a local copy up and running follow these simple example steps.
+### Linter Tests
 
-### Prerequisites
+- `npx hint .`
+- `npx stylelint "**/*.{css,scss}"`
+- `npx eslint .`
 
-### Setup
+### Prerequisites
 
-### Install
+- Have a local version control like Git. Which is a open source distributed version control system designed for source code management
+- A text editor (e.g Visual Studio Code, Vim, Atom & Sublime.)
+- A web browser (e.g Chrome, Safari, Mozilla Firefox.)
+- Live Server which is a web extension that helps you to live reload feature for dynamic content (PHP, Node.js, ASPNET)
+- Download waves to check project accessibilty
 
-### Usage
+### Getting Started
 
-### Run tests
+To get a local copy up and running follow these simple example steps.
 
-### Deployment
+- npm
+  ```sh
+  npm install npm@latest -g
+  ```
 
+### Installation
 
+1. Clone the repo by running the command
+   ```sh
+    git clone https://github.com/V-Blaze/To_Do_List
+   ```
+2. Open the directory of the project
+   ```sh
+   cd To_Do_List
+   ```
+3. Open the html file
+   ```sh
+   Click and open the html file
+   ```
```

Comment:
- [OPTIONAL] Your README is great and has all the sections from the Microverse template, but this section has some information about another person's project, and it is not clear how to use the application. There is an `index.html` in the `src/` folder, but shouldn't the user use the `npm start` after running the `npm install` inside the project directory? It would be great if you change that. 👍